status-im/status-react
# 🗑 Delete message UI

fixes #12286 

### Summary
Add the option to delete chat message. Also introduce a new UI for message actions.

<details>
  <summary>View screenshot 📸</summary>
  
![Simulator Screen Shot - iPhone 11 - 2021-08-03 at 10 29 33](https://user-images.githubusercontent.com/1925158/127960363-a9a05bd3-6c00-4431-bf70-cfd5b2c6d374.png)

</details>

### Review notes
Messages are critical in terms of perf. Verify that this PR doesn't introduces unintended pitfalls.

#### Platforms
- Android
- iOS

#### Areas that maybe impacted
- all chats
 
##### Functional

- 1-1 chats
- public chats
- group chats

### Steps to test
- Send a message (public or private)
- Long press on a message sent by you to see message actions
- You should see a "Delete" option along with other options
- Clicking delete hides the message in the db
- Deleting doesn't affect the receiver's database

status: WIP
